src/App.js

```diff
@@ -5,14 +5,23 @@ import ProductsContainer from "./containers/ProductsContainer";
 import VehiclesContainer from "./containers/VehiclesContainer";
 import CommentsContainer from "./containers/CommentsContainer";
 import CreateThingsContainer from "./containers/CreateThingsContainer";
+
+// import { productLoadStart } from "./actions";
+// import {connect} from "react-redux";
+
+
 class App extends Component {
   constructor() {
     super();
     this.state = {users: []};
   }
   componentDidMount() {
-
+    this.props.loadContacts();
+    this.props.loadVehicles();
+    this.props.loadComments();
+    this.props.loadProducts();
```

Comment:
This works, better would be though to trigger the loading process from the individual components. That way you can reuse them anywhere in your application and don't create this dependency.